include/fizzy/fizzy.h

```diff
@@ -250,6 +255,9 @@ typedef struct FizzyImportedGlobal
 /// @param  error               Pointer to store detailed error information at. Can be NULL if error
 ///                             information is not required.
 /// @return                     true if module is valid, false otherwise.
+///
+/// @note   FizzyError::code must be ::FizzySuccess if function returns `true` and must not be
```

Comment:
Does this not sound like it's a requirement to a user's input?

Reply:
Yeah I think so. Would suggest something like "FizzyError::code will be ::FizzySuccess if functions...".